specification/containerregistry/resource-manager/readme.md

```diff
@@ -260,21 +260,22 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-containerregistry
 
 ``` yaml $(java) && $(multiapi)
 batch:
-  - tag: package-2016-06-preview
+  - tag: package-2018-02-preview
   - tag: package-2017-10
   - tag: package-2017-06-preview
   - tag: package-2017-03
+  - tag: package-2016-06-preview
 ```
 
-### Tag: package-2016-06-preview and java
+### Tag: package-2018-02-preview and java
 
-These settings apply only when `--tag=package-2016-06-preview --java` is specified on the command line.
+These settings apply only when `--tag=package-2018-02-preview --java` is specified on the command line.
 Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.

```

Comment:
Can you update  `--azure-libraries-for-java` to `azure-libraries-for-java-folder`? There are several other places using the name.